### ISSUE TITLE
[Enhancement]:  support for .pls files

### ISSUE THREAD

**Issue by GamerClassN7** [advplyr/audiobookshelf#3047]
### Type of Enhancement

None

### Describe the Feature/Enhancement

Would be possible to add support for *.pls files inside audio book directory  ? many times tehere is correct order of Track saved in this file provider by distributor of  audio books, but now they are not supported do many times audio track are in strange order

### Why would this be helpful?

if supported upload of audio boos would be almost one click :)

### Future Implementation (Screenshot)

Example format 
![image](https://github.com/advplyr/audiobookshelf/assets/22167469/019a2a50-2935-4357-bd2a-cd5ca3b0d9d5)

### Audiobookshelf Server Version

2.10.1

**Comment by advplyr** [comment]
I've never seen this format before. Do you know where these come from? Like what book distributor uses these?

**Comment by GamerClassN7** [comment]
> I've never seen this format before. Do you know where these come from? Like what book distributor uses these?

Almost any in my country Audiotek.com/ alza.cz most of media players supports this files. basically it is jast list of trask.
files in correct order 

https://en.wikipedia.org/wiki/PLS_(file_format)

**Comment by advplyr** [comment]
Hm ok, the track list in the screenshot you shared would have been pulled into Abs in the correct order going just off the filename

**Comment by GamerClassN7** [comment]
> ck list in the screenshot you shared would have been pulled i

Yeah, most of the time but for example this audio book imports in backwards order for some season, even that track metadata were correct in track file :/

It would be really nice if support for this file could be implemented :)
